extensions/aria_extension_tosca/simple_v1_0/modeling/capabilities.py

```diff
@@ -162,6 +164,30 @@ def convert_capability_from_definition_to_assignment(context, presentation, cont
     return CapabilityAssignment(name=presentation._name, raw=raw, container=container)
 
 
+def merge_capability_definition(context, presentation, capability_definition,
+                                from_capability_definition):
+    raw_properties = OrderedDict()
+
+    # Merge properties from type
+    from_property_defintions = from_capability_definition.properties
+    merge_raw_parameter_definitions(context, presentation, raw_properties, from_property_defintions,
+                                    'properties')
+
+    # Merge our properties
+    merge_raw_parameter_definitions(context, presentation, raw_properties,
+                                    capability_definition.properties, 'properties')
```

Comment:
if i get this right, capability_definitions is the "global" definition of this capability, while "from..." if the definitions for this instance. 
and we first apply our own definitions and only then try and merge the global one - while maintaining the idea that the more specific definitions will be the ones to hold?